### DISCUSSION TITLE
problems with padding inside portal

### DISCUSSION THREAD

**Discussion by anovazzi1** [glideapps/glide-data-grid#623] [Q&A]
![image](https://user-images.githubusercontent.com/70824914/215594512-a607bbad-2124-4bff-b8be-50dda5520aa9.png)
my text cell is getting this weird padding, have anyone else faced this before?
it occurs after i set row height dynamically to each row based on i's content:
        `rowHeight={(index) => {`
          `let height = 40;`
          `columns.map((col, idxCol) => {`
            `if(col.dataType === dataTypes.TEXT) {`
              `let newHeight = 10 + measureTextHeight(data[index][col.title].toString(), col.width-5, 14);`
              `height = (height < newHeight ? newHeight : height);`
            `}`
            `return height;`
        `})`
        `return height;`
        `}}`

**Comment by jassmith** [comment]
Ah this is due to the wrapping. I will fix!

**Reply by jassmith** [reply]
Can you file this as a bug please for trakcing purposes?

**Reply by anovazzi1** [reply]
of course, thanks a lot
